pdtable/io/_excel_openpyxl.py

```
@@ -50,17 +86,19 @@ def write_excel_openpyxl(tables, path, na_rep, style, num_blank_rows_between_tab
         for t in tabs:
             # Keep track of table dimensions for formatting as tuples (num rows, num cols, transposed)
             table_dimensions.append((len(t.df), len(t.df.columns), t.metadata.transposed))
-            _append_table_to_openpyxl_worksheet(t, ws, num_blank_rows_between_tables, na_rep)
+            _append_table_to_openpyxl_worksheet(t, ws, sep_lines, na_rep)
 
-        if style:
-            _format_tables_in_worksheet(ws, table_dimensions, num_blank_rows_between_tables)
+        if styles:
+            styles = DEFAULT_STYLE_SPEC if styles is True else styles
```

Should there be a validation of the `styles` dictionary? Or should it simply "not style" if the dictionary is not setup correctly?

---

Hm, good question. Right now it looks for styles in the dict; and applies them if it finds them. If there's a typo in the style keys (e.g. `fnot` instead of `font`, or `taaaable_name` instead of `table_name`), then that part just gets ignored, just as irrelevant dictionary keys are (e.g. an `'hello_world'` entry can be there, but it gets ignored). 

If, however, the _value_ is bad (e.g. `'color': 'my_favorite_color  # <<< not a valid RGB hex code!' or `'alignment': {'horizontal': 'bogus'}  # <<< unknown horizontal alignment!`), then this will fail hard. We could add some validation at the input, but covering all possible valid font, fill, alignment, etc. property values would be a daunting amount of work. 

I think the middle ground here may be to do better error handling. E.g. try applying horizontal alignment, catch any obscure error thrown by openpyxl, and raise our own, more intelligible error. I'll do that. 

If

---

Done. Now the user will get a friendly error message from which they can infer where their style spec has gone awry. Will push soon.